opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -422,7 +423,13 @@ export const stixCoreObjectImportDelete = async (context, user, fileId) => {
     await deleteFile(context, user, fileId);
     // Patch the updated_at to force live stream evolution
     const files = (previous.x_opencti_files ?? []).filter((f) => f.id !== fileId);
-    await elUpdateElement({ _index: previous._index, internal_id: entityId, updated_at: now(), x_opencti_files: files });
+    await elUpdateElement({
+      _index: previous._index,
+      entity_type: previous.entity_type,
```

Comment:
Why adding entity_type here ?

Reply:
I think it's because we need `entity_type` in `prepareDataFromSchemaDefinition`